config.go

```diff
@@ -135,6 +137,11 @@ func (c *Config) getNodeInfo() error {
 	// Get Mesos master/agent ID
 	mid, err := node.MesosID(nil)
 	if err != nil {
+		// It's possible that we encountered an SSL error
+		if useSSL {
+			log.Warnf("Received an error when attempting to get Mesos ID: %q; falling back to HTTP", err)
```

Comment:
Oh, nevermind, I saw that for strings, that this quotes the string. Totally makes sense.